custom/icds_reports/models/aggregate.py

```diff
@@ -862,6 +862,7 @@ class DailyAttendance(models.Model):
     class Meta:
         managed = False
         db_table = 'daily_attendance'
+        unique_together = ('supervisor_id', 'case_id')
```

Comment:
The primary key here is `doc_id` not `case_id`, please check the others.

Reply:
Updated https://github.com/dimagi/commcare-hq/pull/23544/commits/51d7c8feeecb054cc21fb1e081d0ee763e53692f